packages/mwp-app-render/src/components/connectedWithMatchMedia.test.jsx

```diff
@@ -0,0 +1,38 @@
+import jsdom from 'jsdom';
```

Comment:
just as a heads up, as we move more React modules into the platform, I've been thinking we might need to set up a separate test run for things that are designed to be run in the browser - currently Jest is [configured to run](https://github.com/meetup/meetup-web-platform/blob/master/jest.config.json#L15) in a `node` environment, without the jsdom overhead, but we could have a parallel Jest config that targeted test modules with a `.test.dom.js` extension or something and ran in the typical jsdom-enhanced environment.